include/proxy-wasm/wasm.h

```diff
@@ -15,6 +15,8 @@
 
 #pragma once
 
+#include <string.h>
```

Comment:
Do we need to use the old style c function? If not, I think `#include <string>` is enough.

Reply:
We use memcpy which for historical reasons I suppose comes from string.h.

Reply:
Ok, I think using `string.h` makes sense